targets/f7/inc/FreeRTOSConfig.h

```diff
@@ -30,12 +30,9 @@
 #define configMAX_TASK_NAME_LEN (32)
 
 /* Run-time stats - broken ATM, to be fixed */
```

Comment:
not relevant anymore

Reply:
relevant because it is a part of TCB and default is 16.